river/sync.go

```diff
@@ -28,6 +28,16 @@ const (
 	// for the mysql int type to es date type
 	// set the [rule.field] created_time = ",date"
 	fieldTypeDate = "date"
+	// for the mysql lat, lon two field to es geo_point type
+	/*
+		 set the [rule.field]
+			 lat = "location,geo_point.lat"
+			 lon = "location,geo_point.lon"
+	*/
+	fieldTypeGeopointLat = "geo_point.lat"
+	fieldTypeGeopointLon = "geo_point.lon"
+
+	fieldTypeBoolean = "bool"
```

Comment:
is this type associated with geo type?
If not, I suggest using another PR to support it.